### ISSUE TITLE
Module incompatibilities break the term customizer

### ISSUE THREAD

**Issue by Quentinchampenois** [OpenSourcePolitics/decidim-nyc#58]
#### Description

As an admin, when I try to add a new translation in the term customizer panel the AJAX request returns a 500 error code. 

#### Stacktrace

```
Completed 500 Internal Server Error in 8641ms (ActiveRecord: 0.0ms | Allocations: 59096)

NoMethodError (undefined method `translations' for nil:NilClass):
  
decidim-module-term_customizer (f0d720710822) lib/decidim/term_customizer/translation_directory.rb:46:in `backend_translations'
decidim-module-term_customizer (f0d720710822) lib/decidim/term_customizer/translation_directory.rb:18:in `translations'
decidim-module-term_customizer (f0d720710822) lib/decidim/term_customizer/translation_directory.rb:26:in `translations_by_key'
decidim-module-term_customizer (f0d720710822) lib/decidim/term_customizer/translation_directory.rb:22:in `translations_search'
decidim-module-term_customizer (f0d720710822) app/controllers/decidim/term_customizer/admin/add_translations_controller.rb:43:in `search'
actionpack (6.0.6.1) lib/action_controller/metal/basic_implicit_render.rb:6:in `send_action'
actionpack (6.0.6.1) lib/abstract_controller/base.rb:195:in `process_action'
actionpack (6.0.6.1) lib/action_controller/metal/rendering.rb:30:in `process_action'
actionpack (6.0.6.1) lib/abstract_controller/callbacks.rb:42:in `block in process_action'
activesupport (6.0.6.1) lib/active_support/callbacks.rb:112:in `block in run_callbacks'
```

#### Informations

* Decidim `v0.26.7` 
* [Notion card](https://www.notion.so/opensourcepolitics/NYC-Bug-Term-Customizer-4d707cb418e8480680e02ed2cd9da968?pvs=4)

#### Causes

There is an incompatibility between [decidim-module-term_customizer](https://github.com/mainio/decidim-module-term_customizer) and [decidim-module-ptp](https://github.com/Pipeline-to-Power/decidim-module-ptp) 

They both redefine the I18n backend : 
* https://github.com/mainio/decidim-module-term_customizer/blob/f0d720710822f1231ea249dd71f978143d38a6c4/lib/decidim/term_customizer/engine.rb#L10
* https://github.com/Pipeline-to-Power/decidim-module-ptp/blob/18c7422633b743ff6fe63e1dda90bb3480d741a8/decidim-l10n/lib/decidim/l10n/engine.rb#L17

**Comment by Quentinchampenois** [comment]
Yes we could not call the gem but in the current module `decidim-l10n` is defined as a required dependency so the gem will add it by default.

This is why I had to fork and remove this dependency

See original : https://github.com/Pipeline-to-Power/decidim-module-ptp/blob/18c7422633b743ff6fe63e1dda90bb3480d741a8/decidim-ptp.gemspec#L24

**Comment by Quentinchampenois** [comment]
My bad you are right, we can load the exhaustive list of wanted gems in the Gemfile and it seems to fix the issue, thank you I remove the fork !

**Comment by Quentinchampenois** [comment]
Fixed by #59
